helm-repositories/kaptain/kaptain-repo.yaml

```diff
@@ -7,6 +7,6 @@ metadata:
   labels:
     kommander.d2iq.io/dkp-airgapped: supported
 spec:
-  url: "${helmMirrorURL:=http://kaptain-helm-mirror.kommander.svc}"
+  url: "${helmMirrorURL:=https://mesosphere.github.io/kaptain-charts}"
```

Comment:
do we introduce this change just to tag for v2.1.0 and then revert it?
I'm wondering how the [local dev chartmuseum HR approach](https://github.com/mesosphere/kaptain/blob/753fe0ebd446373dd51aa384611b9d3bedc1ec85/manifests/kaptain-applications-git-repo.yaml#L13) would work if this is the up-to-date version of the main branch.

Reply:
Yes, I don't like this approach, but we need to update it each time we create the release. I'm planning to revisit this as a part of release automation improvements. If you need to test your changes, you might need to create a branch here with this URL pointing to the chartmuseum and then update   https://github.com/mesosphere/kaptain/blob/main/manifests/kaptain-applications-git-repo.yaml to use this branch.